samples/bookinfo/src/productpage/uid_entrypoint

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 if ! whoami &> /dev/null; then
```

Comment:
Hah, I guess you had problems with this?

Reply:
yup.